[Doc Feature] Format bot messages with enriched Markdown (#14376)
MicrosoftDocs/msteams-docs: msteams-platform/bots/streaming-ux.md

3. **Format the final streamed message**:

>     Text messages with extended Markdown (using `textFormat: "extendedMarkdown"`) are formatted as they're streamed. Extended Markdown content such as math equations, callouts, fenced blocks, images, and inline Adaptive Cards render progressively at safe boundaries. For standard Adaptive Cards attachments or rich HTML, formatting is applied once the final message is complete. The bot can send attachments only in the final streamed chunk.

Reviewer: we dnt support math equations, callouts, inline adaptive cards yet. pls remove those from the docs.

>     Using AI SDK, text messages and simple markdown (including extended Markdown with `extendedMarkdown`) can be formatted while they're being streamed. Extended Markdown content such as math equations, callouts, fenced blocks, images, and inline Adaptive Cards render progressively at safe boundaries. However, for standard Adaptive Cards attachments, images, or rich HTML, the formatting can be applied once the final message is complete. The bot can send attachments only in the final streamed chunk.

Reviewer 2: ```suggestion
Text messages and markdown (including extended Markdown with `extendedMarkdown`) are formatted as they're streamed.
    
Most extended Markdown content such as math equations, callouts, fenced blocks, images, and inline Adaptive Cards render progressively at safe boundaries. For standard Adaptive Cards attachments, images, or rich HTML, the formatting is applied once the final message is complete. The bot can send attachments only in the final streamed chunk.
```

Needs a check to make sure it's still precise. "Using AI SDK" (Teams SDK?) didn't seem to add anything.

This seems to say that regular markdown formats while streaming too, is that correct? The language in the new "Streaming with extended Markdown" section of the formatting article kind of implies it doesn't; the article doesn't say anything about streaming otherwise.

Author: Updated the content to say that only extended markdown supports.
